Omnibus.md

```diff
@@ -2073,12 +2087,13 @@ is complete.
 
 ### Advanced Regeneration ### {#advanced-regeneration}
 
-An advanced regeneration is a regeneration from death [(]{.new}or scalping[, if
-specified in the spell description)]{.new} which begins when the character dies
-(or is scalped[,]{.new} if it brings a character back from scalping). Each time
-they die, the spellcaster may choose to double the length of their regeneration
-(to 240 seconds, or 480 if diseased). The regeneration is not considered used if
-the target is raised before their regeneration is complete.
+An advanced regeneration is a regeneration from death [(]{.new}or 
+[soul loss, if specified in the spell description)]{.new} which begins when 
+the character dies (or is [rendered soulless,]{.new} if it brings a character 
+back from [losing their soul]{.new}. Each time they die, the spellcaster may 
```

Comment:
`back from [losing their soul)]{.new}. Each time they die, the spellcaster may `